server/json-logger.h

```diff
@@ -65,6 +65,7 @@ class JsonLogger : vk::not_copyable {
   volatile sig_atomic_t json_traces_count{0};
 
   int64_t release_version_{0};
+  std::string_view hostname_{""};
```

Comment:
do you mean this?
```suggestion
  vk::string_view hostname_{""};
```

Reply:
yes, you're right. I must have written `std::string_view` out of habit when I wrote the code here